.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           key: gradle-cache-{{ checksum "manage-accounts-specs/build.gradle" }}
       - run:
           name: Run Integration Tests
-          command: ./gradlew test
+          command: ./gradlew check
```

Comment:
from previous PR - `check` runs all the tests (not that in practise there is any difference)